packages/okta-vue/test/e2e/harness/scripts/downloadChromeDriver.sh

```diff
@@ -1,18 +1,25 @@
 # Install ChromeDriver.
 OS="`uname`"
 
+if [ $# -eq 0 ]
```

Comment:
I'd feel better if we just tested for eq "legacy", then this code is more self-documenting

Reply:
Maybe even better, can pass in the version.  `./downloadChromeDriver 2.45`